src/support/lockedpool.cpp

```diff
@@ -27,7 +27,6 @@
 #include <iomanip>
 #include <iostream>
 #endif
-#include <utility>
```

Comment:
27151: not really a duplicate in our case - consider backporting 26924 which adds the second one or drop this backport

Reply:
dropped